swedbank-pay-payment-menu.php

```diff
@@ -24,7 +24,7 @@
 
 
 defined( 'ABSPATH' ) || exit;
-define( 'SWEDBANK_PAY_VERSION', '4.2.1' );
+define( 'SWEDBANK_PAY_VERSION', '4.2.2' );
```

Comment:
Issue: This should also be updated from the main branch before we do a merge to avoid issues after the latest release. Same with row 10 above.